### ISSUE TITLE
Could not save API Key

### ISSUE THREAD

**Issue by Deozaan** [itchio/butler#27]
I tried pushing a change using Butler, authenticated the Wharf scope to use my Itch account, and got this error:

```
Authenticated successfully! Saving key in C:\Users\[USERNAME]\.itch\butler_creds...

Could not save API key: open C:\Users\[USERNAME]\.itch\butler_creds: The system cannot find the path specified.

invalid character '<' looking for beginning of value
```

This is on Windows 10 Pro x64.

Why is it trying to save there? How come it's not trying to save the file in %APPDATA%\Roaming\itch with the rest of the Itch client and Itch related files?

The weird thing is that I'm pretty sure I already successfully authenticated Butler to use my Itch account. But maybe Butler and Wharf need separate authentications?

**Comment by Deozaan** [comment]
I tried again using _username/game:channel_ instead of _game_id:channel_ and even though it still had an error saving the API key, it successfully pushed the build. Maybe there's another error involved with using game_id as well?

**Comment by Deozaan** [comment]
I just noticed that was already reported and marked as fixed in #24. Sorry. I did try searching before reporting but nothing showed up, probably because it automatically added "status:Open" to the search.

I guess I was running an older version of Butler. I'm now running v0.13.0 and it's not even asking me to authenticate anymore, which I assume means it managed to save the credentials somewhere on my computer.
